include/rgl/api/core.h

```diff
@@ -713,6 +713,15 @@ RGL_API rgl_status_t rgl_node_raytrace_configure_non_hits(rgl_node_t node, float
  */
 RGL_API rgl_status_t rgl_node_raytrace_configure_mask(rgl_node_t node, const int8_t* rays_mask, int32_t rays_count);
 
+/**
+ * Modifies RaytraceNode to set beam divergence.
+ * Beam divergence is used to calculate the beam width at the distance of hit point.
+ * Setting beam divergence > 0.0f is required to use query for multi-return results.
```

Comment:
Can I maybe set beam divergence to 0.0f through this call, in order to disable it? This partially refers to line 949 in apiCore.cpp, where you check for `divergence >= 0.0f`.

Reply:
Yes, that was my intention. I'll mention this in the documentation, thanks.